server/integration/testsuite/src/test/java/org/infinispan/server/test/jcache/JCacheStateAwarenessIT.java

```diff
@@ -0,0 +1,212 @@
+   }
+
+   /**
+    * Test enableStatistics command on JCache API <br>
+    * <br>
+    * Fix pending in <b>ISPN-7095</b>, needs to be uncommented when implemented
+    * 
+    * @throws ReflectionException
+    * @throws MalformedObjectNameException
+    * @throws InstanceNotFoundException
+    * @throws IntrospectionException
+    * @throws MBeanException
+    * @throws AttributeNotFoundException
+    */
+   // @Test
+   public void testEnableStatistics() throws IntrospectionException, InstanceNotFoundException,
+         MalformedObjectNameException, ReflectionException, AttributeNotFoundException, MBeanException {
+      String mBeanName = getMBeanName(enableStatisticsTestCacheName);
+
+      try {
+         cacheManager.enableStatistics(enableStatisticsTestCacheName, false);
+      } catch (NullPointerException npe) {
+         npe.printStackTrace();
+         fail("NullPointerException thrown while attempting to disable statistics");
+      }
+
+      assertTrue("Statistics register as enabled after disabling",
+            !(Boolean) mBeanServer.getAttribute(new ObjectName(mBeanName), "StatisticsEnabled"));
+
+      try {
+         cacheManager.enableStatistics(enableStatisticsTestCacheName, true);
+      } catch (NullPointerException npe) {
+         npe.printStackTrace();
+         fail("NullPointerException thrown while attempting to enable statistics");
+      }
+
+      assertTrue("Statistics register as disabled after enabling",
+            (Boolean) mBeanServer.getAttribute(new ObjectName(mBeanName), "StatisticsEnabled"));
+
+   }
+
+   /**
+    * Test enableManagement command on JCache API<br>
+    * <br>
+    * Fix pending in <b>ISPN-7095</b>, needs to be uncommented when implemented
+    * 
+    */
+   // @Test
+   public void testEnableManagement() {
+      getMBeanName(enableManagementTestCacheName);
+   }
+
+   /**
+    * Enables management and retrieves canonical name of managed bean
+    * 
+    * @param testCacheName
+    *           name of the cache to enable management on
+    * @return canonical name of the managed bean
+    */
+   private String getMBeanName(String testCacheName) {
+
+      try {
+         cacheManager.enableManagement(testCacheName, true);
+      } catch (NullPointerException npe) {
+         fail("NullPointerException thrown while attempting to enable management");
+      }
+
+      Set<ObjectName> mBeanSet = mBeanServer.queryNames(null, null).parallelStream()
+            .filter(n -> n.getCanonicalName().toUpperCase().contains(testCacheName.toUpperCase()))
+            .collect(Collectors.toSet());
+
+      assertEquals("No mBeans matching the cache found", mBeanSet.size(), 1);
+
+      return mBeanSet.iterator().next().getCanonicalName();
+   }
+
+   /**
+    * Test whether cache is found in cache name list of JCache API, fix
+    * pending<br>
+    * <br>
+    * Fix pending in <b>ISPN-7095</b>, needs to be uncommented when implemented
+    */
+   // @Test
+   public void testCacheNames() {
```

Comment:
This test could have a better name, like "validateConfiguration" or something similar to that...